superset/assets/javascripts/reduxUtils.js

```diff
@@ -64,13 +64,13 @@ export function addToArr(state, arrKey, obj) {
   return Object.assign({}, state, newState);
 }
 
-export function enhancer() {
-  let enhancerWithPersistState = compose(persistState());
+export function enhancer(persist = true) {
+  let enhancerWithPersistState = persist ? compose(persistState()) : compose();
```

Comment:
this var name seems misleading now, since it won't always be instantiated with `persistState`. would it be more clear if we called this var `enhancer` and call the function `initEnhancer`?